CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### 0.3.2
 
+- Deprecate setting the inverse active record association on cache hits. Set IdentityCache.never_set_inverse_association to true to avoid this.
```

Comment:
I think the PR number (#279) should be added at the end to keep the logs consistent with the last two entries.